lib/hitorigoto_reporter/formatter.rb

```diff
@@ -17,6 +17,11 @@ def to_markdown
         .join("\n\n\n")
     end
 
+    def stamps
+      filtered_list, file_comments = filter_list_and_extract_comments
+      filtered_list.map{ |h| h.stamps }.flatten.uniq
```

Comment:
I think `Enumerable#flat_map` and [`&` operator](https://www.reddit.com/r/ruby/comments/2ipgx6/help_understanding_mapjoin/) make your code more smart :yum: